agent_metrics/README.md

```diff
@@ -9,6 +9,8 @@ Get metrics from agent_metrics service in real time to:
 
 **NOTE**: The Agent Metrics check has been rewritten in Go for Agent v6 to take advantage of the new internal architecture. Hence it is still maintained but **only works with Agents prior to major version 6**.
 
+To collect Agent metrics for Agent v6+, use the [Go-expvar check][10] with [the `agent_stats.yaml ` configuration file][9] packaged with the Agent.
+
```

Comment:
Should we break these into v5 and v6  H3 subsections?

Reply:
I don't think so, we should update the go-expvar documentation to take into account this specific usage though.